### PR TITLE
Sync upstream and necessary change for enabling Travis CI

### PR DESCRIPTION
**Pull request by zhouer**

### CODE REVIEW

File: contrib/debian/bitcoin-tx.install [hunk 1]
```diff
@@ -0,0 +1 @@
+usr/local/bin/bitcoin-tx usr/bin
```
Comment (hihiben) [inline contrib/debian/bitcoin-tx.install:1]: 我們bitcoin-tx應該拔掉了

File: contrib/bitcoin-qt.pro [hunk 2]
```diff
@@ -11,7 +11,7 @@ FORMS += \
     ../src/qt/forms/overviewpage.ui \
```
Comment (hihiben) [inline contrib/bitcoin-qt.pro:11]: qt應該都拔了

**Comment by lbyyy** [comment]
pass

**Comment by hihiben** [comment]
pass

**Pull request merged**